include/z64interface.h

```diff
@@ -1,6 +1,8 @@
 #ifndef Z64INTERFACE_H
 #define Z64INTERFACE_H
 
+#include "z64view.h"
```

Comment:
not the only header missing from includes in this file lol, that should get cleaned up eventually (out of scope for this)